gvr-javascript/app/src/main/res/raw/fragment.fsh

```diff
@@ -12,13 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#extension GL_ARB_separate_shader_objects : enable
```

Comment:
These extensions are in every shader in the framework. Are they not something that the framework should be adding since it expects them apparently? When using OpenGL their removal has no effect. If only Vulkan requires them then there is even a stronger case for this to be handled by the fw.